package.json

```diff
@@ -64,35 +64,35 @@
     "jszip": "^3.6.0",
     "lodash.mergewith": "^4.6.2",
     "resize-observer-polyfill": "^1.5.1",
-    "rxjs": "^6.6.6",
+    "rxjs": "^6.6.7",
     "setimmediate": "^1.0.5",
     "tslib": "^2.0.0",
     "uuid": "^8.3.0",
     "web-animations-js": "^2.3.2",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1102.4",
-    "@angular-devkit/schematics": "^11.2.4",
-    "@angular-eslint/builder": "^2.0.0",
-    "@angular-eslint/eslint-plugin": "^2.0.0",
-    "@angular-eslint/eslint-plugin-template": "^2.0.0",
-    "@angular-eslint/schematics": "^2.0.1",
-    "@angular-eslint/template-parser": "^2.0.0",
-    "@angular/cli": "~11.2.4",
-    "@angular/compiler-cli": "^11.2.5",
-    "@angular/language-service": "^11.2.5",
+    "@angular-devkit/build-angular": "~12.0.0-next.9",
+    "@angular-devkit/schematics": "^12.0.0-next.9",
+    "@angular-eslint/builder": "^2.0.2",
+    "@angular-eslint/eslint-plugin": "^2.0.2",
+    "@angular-eslint/eslint-plugin-template": "^2.0.2",
+    "@angular-eslint/schematics": "^2.0.2",
+    "@angular-eslint/template-parser": "^2.0.2",
+    "@angular/cli": "~12.0.0-next.9",
+    "@angular/compiler-cli": "^12.0.0-next.9",
+    "@angular/language-service": "^12.0.0-next.9",
     "@angularclass/hmr": "^2.1.3",
-    "@types/jasmine": "^3.3.16",
+    "@types/jasmine": "^3.6.7",
```

Comment:
Were those manually changed? If so, don't just update the typings without the actual package they describe as they expose more recent API that actually might not be available and throw at runtime.

If some automatic update process changed just the types w/o the actual source package, then it's a bug w/ that process.

Reply:
@damyanpetev They were bumped with `npm update`. Downgrade them if there's issues.